<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServentCard</class>
 <widget class="QWidget" name="ServentCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>182</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="label_attack">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-image: url(:/img/attack.png);</string>
   </property>
   <property name="text">
    <string>填攻击值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_health">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>5</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-image: url(:/img/ServentHP_small.png);</string>
   </property>
   <property name="text">
    <string>填生命值</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
